content/en/blog/_posts/2022-12-XX-kubernetes-release-artifact-signing.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: blog
+title: "Kubernetes 1.26: We're now signing our binary release artifacts!"
+date: 2022-12-6
+slug: kubernetes-release-artifact-signing
+---
+
+**Author:** Sascha Grunert
+
+The Kubernetes Special Interest Group (SIG) Release is proud to announce that we
+graduated the [Signing Release Artifacts Enhancement Proposal][kep] to _beta_.
+
+[kep]: https://github.com/kubernetes/enhancements/issues/3031
+
+Signing artifacts provides end users a chance to verify the integrity of the
+downloaded resource. It allows to mitigate man-in-the-middle attacks directly on
+the client side and therefore ensures the trustfulness of the remote serving the
+artifacts. The overall goal of the Signing Release Artifacts Enchnacment
+Proposal (KEP) is to define the used tooling for signing all Kubernetes related
+artifacts as well as providing a standard signing process for related projects
+(for example in [kubernetes-sigs][k-sigs]).
+
+[k-sigs]: https://github.com/kubernetes-sigs
+
+We already sign all officially released container images from Kubernetes v1.24.
```

Comment:
Maybe mention that this was alpha quality?

Reply:
Done :+1: